site/docs/resources/faq.md

```diff
@@ -100,11 +100,10 @@ You have already made sure to use the minimum number of API calls for the most c
 There are a few things you can do:
 
 1. Read this [article](../advanced/flood) in the docs to gain a basic understanding of the situation.
-2. Use the [`transformer-throttler` plugin](../plugins/transformer-throttler).
-3. Use the [`auto-retry` plugin](../plugins/auto-retry).
-4. Come ask us in the [group chat](https://t.me/grammyjs) for help.
+2. Use the [`auto-retry` plugin](../plugins/auto-retry).
+3. Come ask us in the [group chat](https://t.me/grammyjs) for help.
```

Comment:
I thought the goal of the PQ was to please don’t this when I first saw it.

Reply:
Haha nope, please don't use the throttler 😁